Cargo.toml

```diff
@@ -97,8 +97,7 @@ prost-types = "0.13.4"
 quote = "1.0.37"
 # TODO(RSDK-10202): Update `rand` to latest release
 rand = "~0.8"
-# TODO(RSDK-8996): Upgrade `rcgen` to latest release
-rcgen = "~0.12"
+rcgen = "~0.13"
```

Comment:
Do you still need the `~` now? That was there to effect the pin.

Reply:
changed to 0.13.2 (1.13.3 war yanked)